addon/components/popover-on-element.js

```diff
@@ -1,122 +1,14 @@
-import Ember from 'ember';
-import TooltipAndPopoverComponent from 'ember-tooltips/components/tooltip-and-popover';
```

Comment:
I see below that this file exists. Maybe you could just explain how the architecture of the components has changed with this PR? For example, what inherits from what, what wraps what, why existing components can be used outside of the lazy rendering wrapper, etc.